mpp/include/mpp_util.inc

```diff
@@ -428,15 +428,19 @@ end function rarray_to_char
   !!
   !! This call implies synchronization across the PEs in the current
   !! pelist, of which <TT>pelist</TT> is a subset.
-  subroutine mpp_declare_pelist( pelist, name )
+  subroutine mpp_declare_pelist( pelist, name, commID )
     integer,                    intent(in) :: pelist(:)
     character(len=*), intent(in), optional :: name
+    integer,         intent(out), optional :: commID
     integer :: i
 
     if( .NOT.module_is_initialized )call mpp_error( FATAL, 'MPP_DECLARE_PELIST: You must first call mpp_init.' )
     i = get_peset(pelist)
     write( peset(i)%name,'(a,i2.2)' ) 'PElist', i !default name
     if( PRESENT(name) )peset(i)%name = name
+#ifdef use_libMPI
```

Comment:
If the `use_libMPI` isn't used and someone send in `commID`, it will be uninitialized.  I think you need a `#else` and set  `commID` to some value.

Reply:
Actually, I don't think I need the macro protection here at all.  In the no-mpi case,  commID will be set by what has already been defined for the world/current peset in[ mpp_init](https://github.com/NOAA-GFDL/FMS/blob/main/mpp/include/mpp_comm_nocomm.inc#L52-L63)

Reply:
Because commID is `intent(out)` in this subroutine, it needs to be defined **in** the subroutine.  The different compilers will handle this differently if it's not set in the subroutine (warning, error, set to some undefined value).  If commID were `intent(inout)` then some value specified else where may be used and it wouldn't need to be set.

Reply:
I'm agreeing with you and saying I can simply remove the #ifdef/#endif and keep the actual _if (present)_ clause.